modules/civiremote_case/civiremote_case.routing.yml

```diff
@@ -0,0 +1,25 @@
+civiremote_case.create_form:
+  path: '/civiremote/case/add/{profile}'
+  defaults:
+    _controller: 'Drupal\civiremote_case\Controller\CaseCreateController::form'
+  options:
+    no_cache: TRUE
+    parameters:
+      profile:
+        type: string
+  requirements:
+    _user_is_logged_in: 'TRUE'
+
+civiremote_case.update_form:
+  path: '/civiremote/case/{id}/update/{profile}'
+  defaults:
+    _controller: 'Drupal\civiremote_case\Controller\CaseUpdateController::form'
+  options:
+    no_cache: TRUE
+    parameters:
+      id:
+        type: int
+      profile:
+        type: string
+  requirements:
+    _user_is_logged_in: 'TRUE'
```

Comment:
Is this always required?

Reply:
Actually I cannot imagine a use case where anyone should be able to edit a case without any authorization. If it'd actually be necessary a custom module with a custom route could be used.

Reply:
I think it make sense to add a new permission here. I'll change that.

Reply:
Access is now limited to permission `access civiremote case`.